docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md

```diff
@@ -113,7 +113,142 @@ the mapping file. Let's see how to do it:
 ## Dataflows
 No need to be in the mapping file. StartLeft will detect them and will fill the OTM dataflow automatically detecting
 the custom name, the source, and the destination
-## Threats
-No need to be in the mapping file. StartLeft will detect them and will fill the OTM threats automatically
-## Mitigations
-No need to be in the mapping file. StartLeft will detect them and will fill the OTM mitigations automatically
\ No newline at end of file
+
+## Threats and Mitigations
+No need to be in the mapping file.
+
+StartLeft will extract OTM hreats and threat instances, as well as OTM mitigations
```

Comment:
"StartLeft will extract OTM **threats** and ..."

Reply:
Done!